src/components/PersonRow/PersonRow.tsx

```diff
@@ -0,0 +1,50 @@
+import React from 'react';
+import { PersonName } from '../PersonName';
+
+import { PersonData, Person } from '../../utils/type';
+import './PersonRow.scss';
+
+export const PersonRow: React.FC <PersonData<Person, number, string>> = ({
+  person, index, people,
+}) => {
+  const motherId: Person| undefined = people.find(individ => person.motherName === individ.name);
+  const fatherId: Person| undefined = people.find(individ => person.fatherName === individ.name);
```

Comment:
```suggestion
  const mother: Person | undefined = people.find(individ => person.motherName === individ.name);
  const father: Person | undefined = people.find(individ => person.fatherName === individ.name);
```
Почему `Id`, если там объект?